AmplifyPlugins/Storage/Tests/StorageHostApp/AWSS3StoragePluginIntegrationTests/AWSS3StoragePluginAccessLevelTests.swift

```diff
@@ -49,16 +51,25 @@ class AWSS3StoragePluginAccessLevelTests: AWSS3StoragePluginTestBase {
     func testUploadAndRemoveForGuestOnly() async throws {
         let logger = Amplify.Logging.logger(forCategory: "Storage", logLevel: .verbose)
 
-        let username = AWSS3StoragePluginTestBase.user1.lowercased()
-        let password = AWSS3StoragePluginTestBase.password
+        let username: String
+        let password: String
+        if useGen2Configuration {
+            username = "\(UUID().uuidString)@amazon.com"
+            password = "Pp123!@\(UUID().uuidString)"
+            _ = try await Amplify.Auth.signUp(username: username, password: password)
+        } else {
+            username = AWSS3StoragePluginTestBase.user1.lowercased()
+            password = AWSS3StoragePluginTestBase.password
+        }
+
         let accessLevel: StorageAccessLevel = .guest
 
         do {
             logger.debug("Sign In")
             let result = try await Amplify.Auth.signIn(username: username, password: password)
             XCTAssertTrue(result.isSignedIn)
             let currentUser = try await Amplify.Auth.getCurrentUser()
-            XCTAssertEqual(username, currentUser.username)
+            //XCTAssertEqual(username, currentUser.username)
```

Comment:
This is similar to Auth integration tests. Since updating the Auth backend to use Username as username for sign up, we can now uncomment this. (using email as username would fail the username check here.